services/s3/src/test/java/software/amazon/awssdk/services/s3/checksums/ChecksumValidatingPublisherTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is not the same copyright profile that we are using. Could you update it?

Here's the intelliJ copyright file template [link](https://github.com/aws/aws-sdk-java-v2/blob/master/build-tools/src/main/resources/software/amazon/awssdk/intellij-copyright-profile.xml), or you can copy paste from other files in this repo.

Reply:
Hi Zoe,

I'm not affiliated with Amazon, so I thought putting an Amazon copyright there would not be kosher. Am I wrong? I don't mind doing it -- just seems odd.

Reply:
Done

Reply:
`aws-java-sdk-v2` is under the Apache License 2.0 and we wanted to make sure all the files follow the same style. 

Thank you for updating it!